0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md

```diff
@@ -282,140 +259,6 @@ protocol data.
 not include ILP errors such as "No quote found", only the cases in which the
```

Comment:
"[ILQP](https://github.com/interledger/rfcs/blob/master/0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md) packets" should be "ILP packets"